### PR TITLE
feat: elevation is working properly now

### PR DESCRIPTION
**Pull request by Darsh0307**
﻿GitHub Issue:  close https://github.com/unoplatform/Uno.Themes/issues/1273

## PR Type

What kind of change does this PR introduce?

 Bugfix

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [x] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Comment by Darsh0307** [comment]
Screenshot 2024-01-04 at 12 19 37 PM

File: src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml [hunk 1]
```diff
@@ -206,7 +206,7 @@
 											  Margin="0,0,6,6"
 											  Background="Transparent"
 											  CornerRadius="{TemplateBinding CornerRadius}"
-											  Elevation="6">
+											  Elevation="0">
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml:209]: This is just hiding the problem, we will still have a white border around anything that is elevated. And the FAB should have an elevation set in order to show the shadow, no?

Comment (Darsh0307) [reply src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml:209]: Does this means that white border is elevated to everything  not only FAB buttons.

**Comment by Darsh0307** [comment]
The Issue is related to Uno core.

**Pull request closed without merging**